### PR TITLE
feat: added icons users roles lock

### PR DESCRIPTION
**Pull request by amontalvof**

### CODE REVIEW

File: packages/icons/src/components/lock/index.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Lock = (props) => {
+    const { className } = props;
```
Comment (yvmunayev) [inline packages/icons/src/components/lock/index.js:5]: @amontalvof normally we put the className and style props

File: packages/icons/src/components/lockFilled/index.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const LockFilled = (props) => {
+    const { className } = props;
```
Comment (yvmunayev) [inline packages/icons/src/components/lockFilled/index.js:5]: same before

File: packages/icons/src/components/roles/index.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Roles = (props) => {
+    const { className } = props;
```
Comment (yvmunayev) [inline packages/icons/src/components/roles/index.js:5]: same before

File: packages/icons/src/components/rolesFilled/index.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const RolesFilled = (props) => {
+    const { className } = props;
```
Comment (yvmunayev) [inline packages/icons/src/components/rolesFilled/index.js:5]: same before

File: packages/icons/src/components/users/index.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Users = (props) => {
+    const { className } = props;
```
Comment (yvmunayev) [inline packages/icons/src/components/users/index.js:5]: same before

File: packages/icons/src/components/usersFilled/index.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const UsersFilled = (props) => {
+    const { className } = props;
```
Comment (yvmunayev) [inline packages/icons/src/components/usersFilled/index.js:5]: same before

**Review by yvmunayev — APPROVED** [review]

**Review by LeandroTorresSicilia — APPROVED** [review]

**Pull request merged**